modules/021-cni-cilium/images/bin-cilium/werf.inc.yaml

```
@@ -51,6 +52,8 @@ shell:
   - cd /go/src/github.com/cilium/cilium
   - find /patches -name '*.patch' | xargs git apply --verbose
   - go mod tidy && go mod vendor && go mod verify
+  # it is hack for adding file from patch to installing proccess of cilium build
+  - git add bpf/lib/dhcp.h 
```

```suggestion
  - git add .
```
Maybe? It will be handy for future patches.